### PR TITLE
Use u64 to enter subsidy tier threshold to avoid rounding errors

### PR DESCRIPTION
**Pull request by MartinquaXD**
Since people are voting on very specific tier thresholds we should avoid rounding errors if possible.
Luckily this can be done easily in this case by parsing the cow based tier subsidy threshold as an `u64` which allows a lossless conversion to `U256`.

Note:
Also updated the help text to reflect the contents of what is currently being voted on: [CIP-3](https://forum.cow.fi/t/cip-3-trading-fee-discount-for-cow-holders/325)

### Test Plan
CI

### CODE REVIEW

File: crates/orderbook/src/main.rs [hunk 1]
```diff
@@ -177,10 +177,11 @@ struct Arguments {
     )]
     partner_additional_fee_factors: HashMap<AppId, f64>,
 
-    /// Used to configure how much of the regular fee a user should pay based on their COW balance.
+    /// Used to configure how much of the regular fee a user should pay based on their
+    /// COW + VCOW balance of the current network.
```
Comment (nlordell) [inline crates/orderbook/src/main.rs:181]: nit: Maybe specify that this amount is not in atoms but in base units.

**Review by nlordell — APPROVED** [review]
LGTM. 

One downside is that if we setup a tier for 0.5 COW that we can't represent it with `u64`.

Another a approach would be to use atoms instead (so 1 COW would be specified as `1000000000000000000`), but that is less convenient in terms of configuration than what you propose.

**Comment by MartinquaXD** [comment]
I think `u64` offers the best balance of input range and verbosity. Since a single base unit of COW will probably never be very expensive needing 0.5 COW more or less to qualify for a fee doesn't make a difference for the end user and simplifies things for us.

**Review by vkgnosis — APPROVED** [review]

**Comment by nlordell** [comment]
> I think `u64` offers the best balance of input range and verbosity.

I also agree. Additionally, changing this to take `U256` is fairly small amount of work that we can do if the need arises. I think the current implementation is a good compromise between precision and ease of configuration.

**Pull request merged**